index.js

```diff
@@ -0,0 +1,102 @@
+
+function validateLineup(lineup) {
+  return validatePlayerByTeam(lineup) &&
+    validatePlayerByGame(lineup) &&
+    validatePlayerPosition(lineup) &&
+    validateMinimumLineup(lineup) &&
+    validateSalaryTotal(lineup)
+}
+
+
+
+
+function validatePlayerByTeam(lineup) {
+  return groupBy(lineup, member => member.teamId)
+    .filter((teamGroup) => teamGroup.values.length >= 3)
```

Comment:
I would have used `> 2` here so that it matches the way the rule is written. Just makes it easier to read.